EventsExpress/ClientApp/src/components/stepper/Stepper.js

```diff
@@ -0,0 +1,149 @@
+import React, { Component } from 'react';
+import PropTypes from "prop-types";
+import "./Stepper.css";
+
+export default class Stepper extends Component 
+{
+    constructor() 
+    {
```

Comment:
We have a style, when in JS/CSS files new block symbols should be displayed on the same line of an element.